src/components/button-toggle/button-toggle.scss

```diff
@@ -29,10 +29,43 @@
   }
 }
 
+.carbon-button-toggle--grouped {
+  .carbon-button-toggle__input:checked ~ .carbon-button-toggle__label{
+    border-color: $blue-sky;
+  }
+
+  + .carbon-button-toggle--grouped {
+    margin-left: -1px;
+  }
+
+  label {
+    border-radius: 0;
+  }
+
+  &:first-of-type label {
+    border-radius: 4px 0 0 4px;
+  }
+
+  &:last-of-type label {
+    border-radius: 0 4px 4px 0;
+  }
+
+  &--disabled {
+    border: 1px solid #D8D9DC;
+  }
+}
+
+.carbon-button-toggle--small {
+  label {
+    padding: 5px;
+  }
+}
+
 .carbon-button-toggle__label--disabled,
 .carbon-button-toggle__label--disabled:hover {
   background-color: $carbon-button__background-color--disabled !important;
   color: $carbon-button__color--disabled !important;
+  border: 1px solid $carbon-button__background-color--disabled !important;
```

Comment:
Can you sort the CSS properties please?

Also do we really need the `!important`? I think it'd be better to get rid of it / them.

Reply:
I think the `!important` is added to guarantee that the button is always disabled if that status is set for UX reasons (likewise errors)

I think that this is OK, I personally would never do it in a situation where I was in control of the CSS, but that can't be guaranteed in the contexts used

Reply:
I suspected there was a reason like that.

I don't quite get it though, because the `!important` rules can still be overridden with other `!important` rules with the same or higher specificity.

Reply:
They can, but they will stop any accidental specificity that might come from pseudo-classes or modifiers in the app code, something like `.app-component button {}` - it provides a safe-guard against accidental css errors like that.

IMO this is probably alright and a good use of `!important`

Reply:
I've re-added the !important @toni-leigh @ianoxley

Reply:
I'll agree to disagree on this one :)

Reply:
How would you solve this problem? I prefer not to use `!important`!

Reply:
[The top suggestion here might work](http://morecss.org/) :-P

Reply:
> How would you solve this problem? I prefer not to use !important!

Short of removing all `!important` rules and fixing the app code? Not sure TBH. That could be a right can of worms.

One for the future methinks #1549